Optional/src/main/java/by/solbegsoft/oprional/impl/OptionalMethodImpl.java

```diff
@@ -17,42 +23,33 @@
 
     @Override
     public String userExecutor(User user) {
-        Optional<User> optionalUser = Optional.ofNullable(user);
-
-        if (!optionalUser.isPresent()) {
-            throw new RuntimeException("nullable user");
-        }
-
+        Optional<User> optionalUser =
+                Optional.ofNullable(Optional.ofNullable(user)
+                        .orElseThrow(() -> new RuntimeException("nullable user")));
```

Comment:
1. It looks super complicated and redundant

```
User result =  return ofNullable(user)
           .orElseThrow(() -> new RuntimeException("nullable user");
return result.toString();
```

Reply:
Can I do this? It's correct?

Reply:
My mistake. But I think you understood what I mean :)